legacy_plugins/insight_to_action/README.md

```diff
@@ -0,0 +1,457 @@
+
+```json 
+  "tags": "_lcm_managed_object"
+```
+
+you will add the tag and the final value will be:
+```json
+  "tags": "_lcm_managed_object custom_table"
+```
+Then all you have to do is save the change and the insight is newly tagged.
+
+---
+
+### Metric tag “custom_column”
+#### for Gooddata CN/Cloud
+
+To add tag to the metric you need to perform `PUT` request ([api reference here](https://www.gooddata.com/docs/cloud/api-and-sdk/api/api_reference_all/#operation/updateEntity@Metrics)):
+
+`/api/v1/entities/workspaces/{workspaceId}/metrics/{objectId}`
+
+To get your metric object to perform `PUT` reqeust, you can use the next api ([api reference here](https://www.gooddata.com/docs/cloud/api-and-sdk/api/api_reference_all/#operation/getEntity@Metrics)):
+
+`/api/v1/entities/workspaces/{workspaceId}/metrics/{objectId}`
+
+Then you can use the response of this request as the body of your `PUT` request and add the tag as described above.
+
+You also can fetch all the metrics by using this endpoint ([api reference here](https://www.gooddata.com/docs/cloud/api-and-sdk/api/api_reference_all/#operation/getAllEntities@Metrics)):
+`/api/v1/entities/workspaces/{workspaceId}/metrics`
+
+
+#### for Gooddata Platform
+Next setting will define which column/metric will be replaced by the final button.
+Here the only thing which you need to do is open the metric from the “Manage” section and manually add the tag.
+
+![column](./assets/customcolumn.png) 
+
+---
+
+### Configuration json structure in the metric format
+#### for Gooddata Platform and Gooddata CN/Cloud
+Fill the format with configuration json structure for the button.
+You can set it directly in Analyzer on the metric which you want to replace with the button.
+The details of the json configuration will be described below in another section.
+
+![cusotm oclumn](./assets/column.png)
+
+---
+
+### Dashboard plugin in the selected dashboard
+You need to tell the exact dashboard that exactly on this dashboard the plugin will run.
+Basically it is enough to set it in one workspace/dashboard and then once it is already set you can only copy the 
+metadata part from one dashboard to another one.
+
+The process is similar as is described **Insight tag “custom_table”** section of this document.
+You will open metadata of the dashboard where the plugin is already used, then you will copy the part “plugins”. 
+In the new dashboard where you want to use the plugin, you will again open the metadata, and you will past the “plugins” 
+part which you copied in the previous step.
+
+Then the new dashboard should be set to use the plugin.
+
+---
+
+### Configuration specifics for the GoodData Platform
+
+#### Feature flag dashboardComponentDevRollout
+Set feature flag dashboardComponentDevRollout = true over the domain. This should be able to be set from the domain 
+admin account.
+
+`https://{{domain_url}}/gdc/domains/{{domain_id}}/config`
+
+---
+
+#### Setting of dashboardPluginHosts
```

Comment:
@dremileta we're missing here the changes in readme from the internal repo(https://github.com/gooddata/dashboard-plugins-internal/compare/7a8a0785a12a77cd97515b900c9a0ea48cc5c2d2..d4517836b42e3af71b50f34e928d598873cc2e1c#diff-4f33da63a1c97d1fae32d7380b99456020914c8f69140e32c63f10103d6e130eR124) (which is why we've done them in the first place :)). Please update it here too